core/workspace_svg.ts

```diff
@@ -1253,21 +1253,22 @@ export class WorkspaceSvg extends Workspace implements IASTNodeLocationSvg {
     if (!existingGroup) {
       eventUtils.setGroup(true);
     }
-
     let pastedThing;
-    // Checks if this is JSON. JSON has a type property, while elements don't.
-    if (state['type']) {
-      pastedThing = this.pasteBlock_(null, state as blocks.State);
-    } else {
-      const xmlBlock = state as Element;
-      if (xmlBlock.tagName.toLowerCase() === 'comment') {
-        pastedThing = this.pasteWorkspaceComment_(xmlBlock);
+    try {
```

Comment:
What was throwing an exception here?

Reply:
If the clipboard is corrupted, the XML parser might throw.